src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -259,7 +257,7 @@ describe('DiffExprResults', () => {
       {
         extras: {
           pagination: {
-            limit: newPagination.total, offset: 0, orderBy: 'logFC', orderDirection: 'DESC', responseKey: 0,
+            limit: 100000, offset: 0, orderBy: 'logFC', orderDirection: 'DESC', responseKey: 0,
```

Comment:
Could you set the limit to 1000000? 100000 might not be enough for multi species experiments.

Realistically it will probably never come up, but it might be better to be safe.